static/app/views/settings/components/dataScrubbing/modals/form/attributeField.tsx

```diff
@@ -61,7 +64,9 @@ export default function AttributeField({
       !traceItemAttributeStringsResult.isLoading &&
       !traceItemAttributeStringsResult.error
     ) {
-      setSuggestedAttributeValues(traceItemAttributeStringsResult.attributes);
+      setSuggestedAttributeValues(
+        elideTagBasedAttributes(traceItemAttributeStringsResult.attributes)
```

Comment:
A comment mentioning the attributes that wont be selectable would be nice

Reply:
Yeah, fair, there is a comment on that function explaining the purpose of it. For posterity (I'll add a comment), this limits the attributes you can see when selecting for pii scrubbing, but we _have to_ currently as `tags[]` syntax is strictly invalid. 

We should address this via fixing the `trace item keys` endpoint to emit the stored/relay-esque syntax at some point, instead of frontend hacks.